### PR TITLE
Remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by jessieay**
**WHY**: This is mostly dev-only stuff that Cylinder folks won't use.
Plus, better not to have dev-specific stuff in the Gemfile when
possible.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,48 +24,23 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
...
   gem 'capybara-accessible'
-  gem 'capybara-screenshot'
   gem 'climate_control'
   gem 'haml-lint', require: false
-  gem 'launchy'
   gem 'poltergeist'
   gem 'pry-rails'
   gem 'rspec-rails'
-  gem 'rspec_junit_formatter'
-  gem 'rubocop', require: false
```
Comment (adarsh) [inline Gemfile]: I'm assuming we don't need both this and `rubocop-rails`? 🚔

File: Gemfile [hunk 2]
```diff
@@ -28,48 +24,23 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
...
 group :development do
   gem 'citizen-scripts', git: 'https://github.com/citizencode/citizen-scripts'
-  gem 'guard', require: false
-  gem 'guard-rspec', require: false
-  gem 'listen', '~> 3.0.5'
-  gem 'meta_request'
-  gem 'pivotal_git_scripts'
-  gem 'pry-rescue'
-  gem 'pry-stack_explorer'
-  gem 'spring'
```
Comment (adarsh) [inline Gemfile]: 🌸

File: Gemfile (hunk 1 above)
Comment (jessieay) [inline Gemfile]: yep! `rubocop-rails` includes this

**Comment by jessieay** [comment]
@adarsh I think that if we add the hakiri webook, which getcalfresh uses, that will run `bundler-audit` (or something similar). We have a card for that here: https://trello.com/c/ZPEvNFUi/30-add-hakiri-for-security

**Pull request merged**